srdb-data.csv

```diff
@@ -6602,8 +6602,8 @@ Record_number,Entry_date,Study_number,Author,Duplicate_record,Quality_flag,Contr
 6634,2018-10-23,7927,Wang,,,RM,China,Zhejiang,,CN-ZJ-FERTBC,2013,1,30.23333333,119.7,175,Fertilized,5 t/ha,21,21,Castanea mollissima,Subtropical,Forest,Managed,Deciduous,Aggrading,Ferrasol,Dry,,,29.6,39.7,30.7,15.8,1422,,,,Gas chromatography,10,10,900,,21,1,,516,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 6635,2018-10-25,7942,Wiaux,,,RM,Belgium,,,BE-NN,2010.5,1.5,50.6669,4.6331,,None,,,,None,Temperate,Agriculture,Managed,Deciduous,Aggrading,Loam,Dry,,,5,75,20,9.7,805,,,,IRGA,,,314,,21,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 6636,2018-10-25,7961,Prolingheuer,,,RM,Germany,Cologne,Selhausen,DE-COL,2009.5,1,50.86972222,6.44916667,,None,,,1,Triticum aestivum,Temperate,Agriculture,Managed,Deciduous,Aggrading,Haplic luvisol (silt loam),Dry,,,15,67,18,9.9,698,,,,IRGA,7,5,314,,7,0.33,Exclusion,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-6637,2018-10-29,7966,Wu,,,RM,China,Guangdong,Heshan,CN-GD-2YRCK,2009,2,22.56666667,112.8333333,,Understory Removal,,2,2,Eucalyptus globulus,Subtropical,Agriculture,Managed,Evergreen,Aggrading,Ultisol,Dry,,,,,,22.5,1534,,,,IRGA,5,2,314,,21,1,,380,86,,,,,,,,,,,,,,,,,1337.3,1717.4,,,,,215.55,,,,,,,,,,5.1,,,,
-6638,2018-10-29,7966,Wu,,,RM,China,Guangdong,Heshan,CN-GD-24YRCK,2009,2,22.56666667,112.8333333,,Understory Removal,,24,24,Eucalyptus globulus,Subtropical,Agriculture,Managed,Evergreen,Aggrading,Ultisol,Dry,,,,,,22.5,1534,,,,IRGA,5,2,314,,21,1,,455,57,,,,,,,,,,,,,,,,,420.15,924.24,,,,,309.6,,,,,,,,,,5.1,,,,
+6637,2018-10-29,7966,Wu,,,RM,China,Guangdong,Heshan,CN-GD-2YRCK,2009,2,22.56666667,112.8333333,,Litter manipulation,Understory Removal,2,2,Eucalyptus globulus,Subtropical,Agriculture,Managed,Evergreen,Aggrading,Ultisol,Dry,,,,,,22.5,1534,,,,IRGA,5,2,314,,21,1,,380,86,,,,,,,,,,,,,,,,,1337.3,1717.4,,,,,215.55,,,,,,,,,,5.1,,,,
```

Comment:
Should this be "Thinned"? Hmm.

Reply:
You are right, this should be "Thinned", do you want me change this? Thanks

Reply:
Yes please